pkg/ebpf/c/tracee.bpf.c

```diff
@@ -770,6 +772,17 @@ statfunc bool find_modules_from_module_kset_list(program_data_t *p)
         if (mod_kobj) {
             struct module *mod = BPF_CORE_READ(mod_kobj, mod);
             if (mod) {
+                if (first_mod_name == NULL) {
```

Comment:
Can't you just compare the module address?
Why do you need the address of the name?

Reply:
Totally right. Was too deep with the kobjects. Thanks!